src/frontend/handlers.go

```diff
@@ -476,7 +476,8 @@ func cartSize(c []*pb.CartItem) int {
 }
 
 func renderMoney(money pb.Money) string {
-	return fmt.Sprintf("%s %d.%02d", money.GetCurrencyCode(), money.GetUnits(), money.GetNanos()/10000000)
```

Comment:
The `renderMoney` function was altered because we no longer spell out the currency — we use the currency symbol instead (e.g., "USD" is not "$").